api/pom.xml

```diff
@@ -30,5 +30,11 @@
 			<scope>provided</scope>
 			<version>${atomfeedVersion}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>webservices.rest-omod-common</artifactId>
```

Comment:
Shouldn't this dependency be in omod instead of api?

Reply:
unfortunately we have no api folder in rest-webservices, this would result into whether org.openmrs.module.webservices.rest.SimpleObject is in the right place?

Reply:
Here? https://github.com/openmrs/openmrs-module-sync2/blob/887e90a2cd426e9e57e06a8d9f08ea11e5044d95/api/src/main/java/org/openmrs/module/sync2/client/rest/RestClient.java

Reply:
https://github.com/openmrs/openmrs-module-sync2/pull/58/files#diff-5f480449d6de65f2e563359a6202e2da

Reply:
How about moving that class to the omod? Because having a web dependency in the api does not smell good.

Reply:
that movement would require moving other classes that are used in api. it may even make reviewing this change way more complex